spec/factories/rent.rb

```diff
@@ -1,4 +1,8 @@
 FactoryBot.define do
   factory :rent do
+    from    { Faker::Date.today }
+    to      { Faker::Date.forward(2) }
+    book    { 1 }
+    user    { 1 }
```

Comment:
same as user, `create(:user)` is better than using an ID.

Reply:
Done